tools/homebrew/drafter.rb

```diff
@@ -2,7 +2,7 @@
 
 class Drafter < Formula
   homepage 'http://apiblueprint.org'
-  head 'https://github.com/apiaryio/drafter.git'
+  head 'https://github.com/apiaryio/drafter.git', :tag => 'v0.1.0'
```

Comment:
Having the explicit tag here prevents from installing any commits into master that weren't yet released. So it is somewhat safer. Unless one forgets to change it, that's it.